### DISCUSSION TITLE
How to activate or import TC's auto-generated json files?

### DISCUSSION THREAD

**Discussion by Mkallday** [thunderclient/thunder-client-support#1596] [Q&A]
### Is it not possible to import TC's environment or collection json files?
![image](https://github.com/user-attachments/assets/920829b6-7ab6-41b1-be33-3430b8ca1ce0)
![image](https://github.com/user-attachments/assets/e3169425-49b2-4323-bcda-799d087fab7b)

**Comment by rangav** [comment]
No its not possible to import the TC raw JSON files.

you need to first export collection or environment and then you can import them.
